src/components/Dialogs/Dialogs.js

```diff
@@ -2,20 +2,23 @@ import React from 'react';
 
 import DialogItem from './DialogsItem/DialogsItem';
 import Message from './Message/Message';
+import { updateNewMessageBodyCreator, sendMessageCreator } from '../../redux/state';
```

Comment:
Consider moving this import up, outside of this "local to the file" group.

Reply:
/fxd